src/auth0-session/get-config.ts

```diff
@@ -133,8 +133,6 @@ const paramsSchema = Joi.object({
   issuerBaseURL: Joi.string().uri().required(),
   legacySameSiteCookie: Joi.boolean().optional().default(true),
   routes: Joi.object({
-    login: Joi.alternatives([Joi.string().uri({ relativeOnly: true }), Joi.boolean().valid(false)]).default('/login'),
-    logout: Joi.alternatives([Joi.string().uri({ relativeOnly: true }), Joi.boolean().valid(false)]).default('/logout'),
     callback: Joi.string().uri({ relativeOnly: true }).default('/callback'),
```

Comment:
Why have a default callback URL?

Reply:
Good point 👍 - that was left over from the express code. Will remove that and make the field required.